docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md

```diff
@@ -131,6 +131,34 @@ Stripe,Pay Online with Stripe,en_US
 console data:import glossary
 ```
 
+## Processing refunds
+
+In the default OMS configuration, a refund can be done for an order or an individual item. The refund action is initiated by a Back Office user triggering the `Payment/Refund` command. The selected item enters the `payment refund pending` state, awaiting the response from Stripe.
+
+During this period, Stripe attempts to process the request, which results in success or failure:
+* Success: the items transition to the `payment refund succeeded` state, although the payment isn't refunded at this step.
+* Failure: the items transition to the `payment refund failed` state.
+
+These states serve to track the refund status and inform the Back Office user. Over the following days, Stripe finalizes the refund, causing the item states to change accordingly. For example, previously successful refunds may be declined and vice versa.
+
+If a refund fails, the Back Office user can go to the Stripe Dashboard to identify the cause of the failure. After resolving the issue, the item can be refunded again.
+
+In the default OMS configuration, seven days are allocated to Stripe to complete successful payment refunds. This is reflected in the Back Office by transitioning the items to the `payment refunded` state.
+
 ## Retrieving and using payment details from Stripe
 
 For instructions on using payment details, like the payment reference, from Stripe, see [Retrieve and use payment details from third-party PSPs](https://docs.spryker.com/docs/pbc/all/payment-service-provider/{{page.version}}/base-shop/retrieve-and-use-payment-details-from-third-party-psps.html)
+
+## Sending additional data to Stripe
+
+Stripe accepts to pass metadata in the API calls for additional data. To send additional data to Stripe, you can use the `QuoteTransfer::PAYMENT::ADDITIONAL_PAYMENT_DATA` field. This field is a key-value array that you can use to pass additional data to Stripe.
+
+The metadata field has some limitations you must ensure to be met:
+
+- Key length: max 40 characters
+- Value length: max 500 characters
+- Key-value pairs: max 50 pairs
+
+When you pass the metadata to Stripe, it is stored in the payment object and can be retrieved later. For example, you can use the metadata to store the order ID or any other data you need to pass to Stripe.
```

Comment:
@stereomon Order ID already stored by our App. Can you provide another example?
Also I think it makes sense to write about our "default" metadata (so it these keys will be ignored if provided).

Reply:
Updated. We do not pass default data. This is the first time we passed metadata to the PaymentIntent creation.